test/echo-server.c

```diff
@@ -68,6 +68,24 @@ static void after_shutdown(uv_shutdown_t* req, int status) {
   free(req);
 }
 
+static void dont_linger(uv_handle_t* handle) {
+  struct linger so_linger;
+  uv_os_fd_t socket_fd = 0;
+
+  if (UV_TCP != uv_handle_get_type(handle)) {
+    return;
+  }
+
+  ASSERT(0 == uv_fileno(handle, &socket_fd));
+
+  so_linger.l_onoff = 1;
+  so_linger.l_linger = 0;
+  ASSERT(0 == setsockopt((uv_os_sock_t) socket_fd,
```

Comment:
no problems with this on Windows?

Reply:
This capability is now equivalent to `uv_tcp_close_reset`, so I suppose it should be fine. I've refactored it to use `uv_tcp_close_reset` directly now.